storybook/src/components/Pagination/Pagination.docs.mdx

```diff
@@ -11,3 +11,38 @@ import README from "../../../../packages/css/src/components/pagination/README.md
 <Primary />
 
 <Controls />
+
+## Examples
+
+### Specifying a link template
+
+Use the `linkTemplate` prop to set the `href` attribute for the Pagination links.
+Remember, if you are using URL parameters (like on a paginated search page),
+make sure to include those parameters too.
+
+{/* prettier-ignore */}
+```tsx
+const linkTemplate = (page) => `/?pagina=${page}&categorie=video&thema=bestuur`
+
+return <Pagination linkTemplate={linkTemplate} />
+```
+
+### With a custom link component
+
+You can pass a custom component to render the Pagination links.
+This can be used to integrate Pagination into the routing library for your application.
+For example, you can use Pagination with the Next.js link component like so:
```

Comment:
I feel we should avoid addressing ‘you’ too explicitly. And it seems we can use less words here as well:

```suggestion
Pass a custom link component to use Pagination with your application’s routing library.
Here’s an example with the Link component of Next.js:
```